storage/scylla_table.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLA_TABLE_H_
+#define TA_SCYLLA_TABLE_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "accelerator/errors.h"
+#include "cassandra.h"
+#include "utarray.h"
+
+typedef struct identity_table_s identity_table_t;
```

Comment:
`identity_table_t` is still misleading here. It is hard to understand the purpose of ScyllaDB operations. We might need explicit prefix such as `db_`.

Reply:
Call it a table is inaccurate, it is defined for members in a row of the identity table.
I will rename it as db_identity_row_t,